lumigator/jobs/evaluator/evaluator.py

```diff
@@ -114,6 +115,11 @@ def eval_command(config: str) -> None:
     config_dict = json.loads(config)
     logger.info(f"{config_dict}")
     config_model = EvalJobConfig(**config_dict)
+
+    # NOTE: Temporary solution to avoid API key issues in G-Eval which defaults to calling OpenAI.
+    if "g_eval_summarization" in config_model.evaluation.metrics and (api_key := os.environ.get("api_key")):
+        os.environ["OPENAI_API_KEY"] = api_key
```

Comment:
Can we log an error or something? Seems like it may be worth having something as an "else" here.

Reply:
In the current code `main`, I think the eval stuff would just conk on its own somewhere. 

We can log something at `error` for sure, I'm not super familiar with the job code (even though I've been tinkering recently) so I don't know if you'd just bail out there, or run eval and be able to get the other metrics at least?